src/features/game/expansion/Land.tsx

```diff
@@ -147,20 +163,37 @@ export const Land: React.FC = () => {
         {expansions
           .filter((expansion) => expansion.readyAt < Date.now())
           .map(
-            ({ shrubs, pebbles, terrains, trees, plots, createdAt }, index) => (
+            (
+              { shrubs, pebbles, stones, terrains, trees, plots, createdAt },
+              index
+            ) => (
               <Expansion
                 createdAt={createdAt}
                 expansionIndex={index}
                 key={index}
                 shrubs={shrubs}
                 pebbles={pebbles}
+                stones={stones}
                 terrains={terrains}
                 trees={trees}
                 plots={plots}
               />
             )
           )}
 
+        {gameState.matches("editing") && (
+          <div
+            className="absolute"
+            style={{
+              top: "50%",
+              left: "50%",
+              zIndex: 100,
+            }}
+          >
+            <Placeable />
+          </div>
+        )}
```

Comment:
A placeable can be a building, decoration or NFT. They are hardcoded here to start drag and drop in them middle of the screen.